src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -107,24 +119,19 @@ public function add(array $headers)
      *
      * @return string|string[]|null The first header value or default value if $first is true, an array of values otherwise
      */
-    public function get($key, $default = null, $first = true)
+    public function get($key, $default = null)
```

Comment:
the corresponding annotation should be removed

Reply:
removed